### PR TITLE
fixed issues, changed file name

### PR DESCRIPTION
**Pull request by sagarkarira**
Fixed issue https://github.com/Nithmr/tipsNtricks/issues/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,232 @@
-**Tips And Tricks**
+> A curated list of softawres, browser plugins, tools, tips and websites made by and for the student of NITH and maintained by the same.
```
Comment (Akarshit) [inline README.md:1]: I am pretty sure that this list could useful to students of other college too.

**Comment by Akarshit** [comment]
@sagarkarira Thanks for the effort, this was really needed. Could you make the small change I mentioned so that I can merge this.

**Comment by sagarkarira** [comment]
@Akarshit I made another PR which is already merged by @akshendra . This had some conflicts. Closing this now. I will make the required changes in another PR.

**Comment by Akarshit** [comment]
@sagarkarira Great, thanks :)

**Pull request closed without merging**